constants/routes.constant.js

```diff
@@ -1,296 +1,376 @@
 "use strict";
+
+/**
+ * ===***===***===***===***===***===***===***===***===
+ * ===***===   PLEASE READ BEFORE EDITING    ===***===
+ * ===***===***===***===***===***===***===***===***===
+ *
+ * If you are adding a route to this list, update this number
+ * next avaiable createFromTime value: 165
+ *
+ * If you are deleting a route from this list, please add the ID to the list of 'reserved' IDs,
+ * so that we don't accidentally assign someone to a given ID.
+ * reserved createFromTime values:
+ */
+
 const Constants = require("./general.constant");
+const mongoose = require("mongoose");
 
 const authRoutes = {
     login: {
         requestType: Constants.REQUEST_TYPES.POST,
-        uri: "/api/auth/login"
+        uri: "/api/auth/login",
+        _id: mongoose.Types.ObjectId.createFromTime(100)
```

Comment:
Why does this start at 100?

Reply:
I'm implicitly reserving `_id` values `1...99` for user roles. I feel it's really unlikely that we have 100 different user roles, but I wanted to give enough space :)